.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+	"singleQuote": true,
```

Comment:
Can you switch this to using double quotes? I prefer that for strings and single ones for characters.

Reply:
I set Prettier to defaults, which is double quotes, so this is done.